submissions/denismaysov/html-css-popup/index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+    <title>HTML-CSS-Hooli-PopUP</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <nav class="header__menu">
+          <ul class="header__list_items">
+            <li class="header__list_item">
+              <a href="#" class="header__link link focus_item">Mail</a>
+            </li>
+            <li class="header__list_item">
+              <a href="#" class="header__link link focus_item">Images</a>
+            </li>
+            <li class="header__list_item">
+              <input
+                type="checkbox"
+                id="popup_checkbox"
+                class="header__popup_checkbox"
+              />
+              <label for="popup_checkbox" class="header__popup_label">
+                <img
+                  src="assets/popup-button.png"
+                  width="25"
+                  height="25"
+                  alt="Кнопка которая открывает Попап"
```

Comment:
If you want to inform the user you should move this text to `aria-label` attribute of the parent label.